### PR TITLE
docs: Publish v2.3 documentation

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

### CODE REVIEW

**Review by stijnmoreels — APPROVED** [review]

File: docs/versioned_docs/version-v2.3/01-index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Arcus Observability"
+layout: default
+slug: /
+sidebar_label: Welcome
+---
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.Observability.Correlation?includePreReleases=true)](https://www.nuget.org/packages/Arcus.Observability.Correlation/)
```
Comment (stijnmoreels) [inline docs/versioned_docs/version-v2.3/01-index.md:8]: Should probably be the fixed version here, no?

File: docs/versioned_docs/version-v2.3/02-Features/correlation.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+- Transaction Id - ID that relates different requests together into a functional transaction.
+- Operation Id - Unique ID information for a single request.
+- Operation parent Id - ID of the original service that initiated the request.
+
+## Installation
+
+This feature requires to install our NuGet package
+
+```shell
+PM > Install-Package Arcus.Observability.Correlation
```
Comment (stijnmoreels) [inline docs/versioned_docs/version-v2.3/02-Features/correlation.md:19]: Should include the version, no?

File: docs/versioned_docs/version-v2.3/01-index.md (hunk 1 above)
Comment (tomkerkhove) [reply docs/versioned_docs/version-v2.3/01-index.md:8]: Added to #235

File: docs/versioned_docs/version-v2.3/02-Features/correlation.md (hunk 2 above)
Comment (tomkerkhove) [reply docs/versioned_docs/version-v2.3/02-Features/correlation.md]: Docs added to #235

**Pull request merged**